components/Form/CheckInForm.tsx

```diff
@@ -6,9 +6,57 @@ import checkInUser from '../../lib/checkInUser';
 import styles from '../../styles/CheckInForm.module.css';
 import Image from 'next/image';
 import userIcon from '../../public/userIcon.png';
+import warningIcon from '../../public/warning.png';
 import Link from 'next/link';
 import handleOffline from '../../lib/handleOffline';
 
+import Lottie from 'react-lottie';
+import confirmationAnimation from '../../lottie/confirmation.json';
+import errorAnimation from '../../lottie/error.json';
+import loadingAnimation from '../../lottie/loading.json';
+
+function getDate() {
```

Comment:
This function does not seem to be in the same design as the rest of the project.
It should be similar to
```js
const getDate = () => {
const current = new Date();
return `${current.getDate()}/${current.getMonth() + 1}/${current.getFullYear()}, ${current.getHours()}:${current.getMinutes()}`;
}
```

Reply:
Extra info:
Arrow functions: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions
Template literals: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals